components/skeleton/skeleton.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import useTheme from "../use-theme";
+
+interface Props {
+    width?: number
```

Comment:
The `width` and `height` props should have the `scale` provided.

Reply:
Exported component with `withScale` HOC